### DISCUSSION TITLE
Why the minimum clickable area is 48 px?

### DISCUSSION THREAD

**Discussion by ptu14** [GoogleChrome/lighthouse#14864] [Q&A] [answered]
Hi,

why the minimum clickable area is [48x48](https://github.com/GoogleChrome/lighthouse/blob/main/core/audits/seo/tap-targets.js#L45)?

[WCAG 2.1 (level AAA) target-size](https://www.w3.org/TR/WCAG21/#target-size) it specifies the minimum value as 44x44 CCS pixels.

[draft WCAG 2.2 (level AA) target-size](https://www.w3.org/TR/WCAG22/#target-size-minimum) goes even further and specifies it as 24x24. But it's still a draft for now.

from what i understand, your value is subjectively set and has nothing to do with WCAG?

does it just mean if you want a higher google search result, do larger clickable areas than WCAG dictates?

**Comment by paulirish** [comment, marked as answer]
Most of the history of the impl can be seen in https://github.com/GoogleChrome/lighthouse/issues/4358

I'm not sure where the magical 48x48 number comes from. Decent chance it tracks back to Android's UI Guidelines via old PSI's rules: https://web.archive.org/web/20141209032129/https://developers.google.com/speed/docs/insights/SizeTapTargetsAppropriately
